sections/deckbuilding.tex

```diff
@@ -291,32 +340,56 @@ \subsection*{\pagetarget{Artifact}{Artifact} Cards}
   \vspace*{\fill}
 \end{center}
 
-\columnbreak
-\subsection*{\pagetarget{Specialty}{Hero Specialty Cards}}
+\pagebreak
 
-Hero Specialty Cards\index{Specialty Card} are gained from \pagelink{Level}{Level ups}.
-Each Main Hero has a unique set of Specialty Cards.
-While some of these Cards have effects which resemble Spell Cards and even make use of \svg{empower}, Specialty Cards are their \textbf{own unique category of Cards}.
-For example, the limit of 1 Spell per Combat Round doesn't apply to Specialty Cards.
+\begin{expansion}{stronghold}
+\subsection*{\pagetarget{Spell Scroll Cards}{Spell Scroll Cards}}
 
-{
-  \medskip
-  \centering
-  \includegraphics[width=0.7\linewidth]{\cards/specialty.png}\\
-  \medskip
-  \footnotesize\textit{A Level 4 Specialty Card, belonging to Catherine the Knight.}
-}
+During setup shuffle 10 of the Spell Scroll Cards into the Artifact Deck.
+Set the remaining 10 Cards aside to form a Spell Scroll Deck.
+Some locations or Scenario effects may call for them later.
+\vspace*{1em}
 
 {
-  \medskip
-  \centering
-  \includegraphics[width=0.7\linewidth]{\cards/specialty2.png}\\
-  \medskip
-  \footnotesize\textit{A Level 1 Specialty Card, belonging to Deemer the Warlock. It uses \svg{empower}, but does NOT belong to any School of Magic, despite the imagery.}
+    \medskip
+    \centering
+    \includegraphics[width=0.7\linewidth]{\cards/test_spellscroll.png}\\
+    \medskip
+    \footnotesize
+    \begin{center}
+         \textbf{\textit{\textcolor{darkcandyapplered}{Spell Scroll Card}}}
+    \end{center}
 }
 
-\clearpage
+\vspace*{1em}
+When you gain a Spell Scroll place it near your Hero Card, take the 2 top Cards from the Spell Deck and place them face down on the Spell Scroll.
+You can look at these Spells at any time and they don't count for your Hand Limit.
+You can only have 2 Spell Scrolls at a time.
+If you gain a third one, draw the new Spells normally and then decide whether you discard the newly gained Spell Scroll or replace one you already have (including all Spells on it).
 
+\medskip
+
+You can play Spells from a Spell Scroll like any other Spell with the following exceptions:
+\begin{itemize}
+    \item They don't count towards your Spell Limit.
+    You can also play a Spell from your hand during this Combat Round.
+    \item You can only use their lowest \svg{empower} effect.
```

Comment:
I'm not a fan of "lowest" here, as the bottom (low) effect on the card is the strongest one. Maybe cheapest? Weakest? Compare it to the original, where it's "lowest power":
<img width="787" height="201" alt="image" src="https://github.com/user-attachments/assets/2da3cc59-f5a7-43d3-bc3e-9fe5c156531b" />